CharlieBackend.Api.UnitTes/ScheduleServiceTests.cs

```diff
@@ -432,24 +420,435 @@ public async Task CreateScheduleAsync_NullRequest_ShouldReturnValidationError()
             result.Should().NotBeNull();
             result.Error.Code.Should().BeEquivalentTo(ErrorCode.ValidationError);
         }
+        
+        [Fact]
+        public async Task GetEventOccurrenceByIdAsync_ExistingId_ShouldReturnExpectedEventOccurrenceDTO()
+        {
+            //Arrange
+            Initialize(validScheduleDTO);
+
+            _eventOccuranceRepositoryMock.Setup(x => x.GetByIdAsync(id)).ReturnsAsync(validEventOccurrence);
+
+            var scheduleService = new ScheduleService(_unitOfWorkMock.Object, _mapper, _scheduledEventFactory);
+
+            //Act
+            var result = await scheduleService.GetEventOccurrenceByIdAsync(id);
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Data.Should().BeEquivalentTo(_mapper.Map<EventOccurrenceDTO>(validEventOccurrence));
+        }
+
+        [Fact]
+        public async Task GetEventOccurrenceByIdAsync_NonExistingId_ShouldReturnNotFound()
+        {
+            //Arrange
+            var scheduleService = new ScheduleService(_unitOfWorkMock.Object, _mapper, _scheduledEventFactory);
+
+            //Act
+            var result = await scheduleService.GetEventOccurrenceByIdAsync(nonExistentId);
+
+            //Assert
+            result.Should().NotBeNull();
+            result.Error.Code.Should().BeEquivalentTo(ErrorCode.NotFound);
+        }
 
         [Fact]
-        public async Task GetEventsFiltered_StartDateBiggerThanFinishDate_ShouldReturnValidationError()
```

Comment:
For some reason (my guess is that it was after making a mistake in conflict resolving) you are completely deleting this unit test in https://github.com/ita-social-projects/WhatBackend/pull/422/commits/c2dabe3bbda02c0b24c1640c89184ce5b48c3eab (that was originally added in https://github.com/ita-social-projects/WhatBackend/pull/452) when you shouldn't.

Reply:
Solved in https://github.com/ita-social-projects/WhatBackend/pull/422/commits/28963aff28f4b032379ac46d424ec91874a8c4d4.
Returned the negative test case of `StartDate` > `FinishDate` back.